vdev/src/git.rs

```diff
@@ -9,6 +9,26 @@ pub fn current_branch() -> Result<String> {
     Ok(output.trim_end().to_string())
 }
 
+pub fn checkout_branch(branch_name: &str) -> Result<()> {
+    let _output = capture_output(&["checkout", "-b", branch_name])?;
```

Comment:
Note that this _creates_ and then checks out the branch.

I wonder if this shouldn't be a little smarter, handling the optional branch create here instead of in the command.

I'm also a little nervous about discarding _all_ output from `git` in these commands, as it makes problem resolution much more difficult.